scalapb-runtime-grpc/src/main/scala/scalapb/grpc/ConcreteProtoMethodDescriptorSupplier.scala

```diff
@@ -0,0 +1,15 @@
+package scalapb.grpc
+
+import com.google.protobuf.Descriptors
+import io.grpc.protobuf.{ProtoMethodDescriptorSupplier, ProtoServiceDescriptorSupplier}
+
+class ConcreteProtoMethodDescriptorSupplier(
```

Comment:
It is sufficient to pass only `methodDescriptor`, since you can get service and file from a `MethodDescriptor`. See https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/Descriptors.MethodDescriptor